maywood/style-editor.css

```diff
@@ -475,6 +475,7 @@ object {
 	background-color: #897248;
 	border-radius: 5px;
 	border-width: 0;
+	text-decoration: none;
```

Comment:
There are a few extra things that showed up when the Sass compiled. I'm guessing these are supposed to be there? But I can prune them out if they're not.